styleguide/source/_patterns/01-molecules/06-components/12-category-highlight-image/category-highlight-image.twig

```diff
@@ -0,0 +1,19 @@
+<div class="layout layout_three_up">
+  <div class="category-highlight layout_three_up-primary">
+    <a class="category-highlight_image-link" href="#"><img class="category-highlight_image" src="{{ src|default('../../assets/images/fpo_3x2.png') }}" alt="Better Patient Health"></a>
+    <a class="category-highlight_title-link" href="#">{% include 'atoms-h4' with { 'content': 'Better Patient Health', 'class': 'b4 category-highlight_title' } %}</a>
```

Comment:
oh and I think instead of the b4 class, you should use the %type-small extend which is in scss > base > generic > _typography. I think megh replaced all the b's with type extensions that are in this file!